pkg/connection/connection.go

```diff
@@ -399,6 +428,24 @@ func (m *Manager) AddSfConnectionFromConfig(connection *config.SnowflakeConnecti
 	return nil
 }
 
+func (m *Manager) AddAwsConnectionFromConfig(connection *config.AwsConnection) error {
+	m.mutex.Lock()
+	defer m.mutex.Unlock()
+
+	if m.Aws == nil {
+		m.Aws = make(map[string]*athena.DB)
+	}
+
+	m.Aws[connection.Name] = athena.NewDB(&athena.Config{
+		Region:          connection.Region,
+		OutputBucket:    connection.QueryResultsPath,
+		AccessID:        connection.AccessKey,
+		SecretAccessKey: connection.SecretKey,
+	})
```

Comment:
we don't really want to do this, that's why my previous implementation used a separate connection fetcher that created a different instance. how would we implement things like S3 sensors if we do this?

Reply:
I think we can change it as soon as we have aws connections for another integration?